conductor/__beta__/job/worker.py

```diff
@@ -73,4 +79,94 @@ def validate_job(self):
         if self.deadline_ssl_certificate is None:
             raise DeadlineWorkerJobError("deadline_ssl_certificate has not been set. This must be the local path to your Deadline client certificate")
         
-        return True
\ No newline at end of file
+        return True
+    
+    def submit_job(self):                
+        
+        deadline_package = conductor.lib.package_utils.get_host_package("deadline", self.deadline_client_version, strict=False)['package']
+        self.software_packages_ids.append(deadline_package)

```

Comment:
Given how the Deadline plugin is built, it wouldn't be possible to execute submit_job() multiple times. However, if the Worker class is used in a different context, it would be possible to call the method multiple times on a single object. I'm pretty sure the functions that merge the package ids would fail (as some of the env variables are 'exclusive'- but I haven't tested. I could cast the list as a set... but that wouldn't avoid the case of multiple (but different version) deadline packages. Do you think this is a case I need to be worried about?